source/dashboards/cards.markdown

```diff
@@ -101,10 +101,12 @@ You can choose to show or hide certain cards based on different conditions. The
 In [sections view](/dashboards/sections/), you can resize cards. Follow these steps:
 
 1. On the **Layout** tab, move the sliders to adjust the card size.
-    - Don't see a **Layout** tab? The picture elements card does not support that yet.
+    - Don't see a **Layout** tab?
+      - It is not available inside nested cards: vertical stack, horizontal stack, and grid card
+      - It is not available on the picture elements card.
   ![Screenshot of the Layout tab in the card dialog](/images/dashboards/card_resize.png)
 
-2. Select **Save**.
+1. Select **Save**.
```

Comment:
CI is complaining here, as the list above is using 1.2.3, and this one 1.1, which is inconsistent.